internal/layer2/announcer.go

```diff
@@ -131,6 +131,15 @@ func (a *Announce) updateInterfaces() {
 	return
 }
 
+// Force a proactive announcement of all IP addresses
+//
+// Proactively send another round of gratuitious ARP / NDP messages.  This may be
```

Comment:
Yes.  Btw, #710 will make it so you don't have to nag me manually on this.  :-)

I also used it to help me find them:

```
$ inv checkpatch
+// Proactively send another round of gratuitious ARP / NDP messages.  This may be
+// detect changes on a per IP basis.  The value is a map of the current
+       // changed since the last time we checked this Service's IP.  If so, we kick off
+       // sending gratuitous ARP / NDP for this Service IP.  This will handle the
+       // IP address.  This happens asynchronously.
ERROR: Found changed lines with 2 spaces after a period.

```

Reply:
LOL. Thank you.